internal/pkg/service/buffer/worker/service/file_test.go

```diff
@@ -1,629 +0,0 @@
-package service_test
-
-import (
-	"context"
-	"fmt"
-	"io"
-	"strings"
-	"testing"
-	"time"
-
-	"github.com/c2h5oh/datasize"
-	"github.com/keboola/go-client/pkg/keboola"
-	"github.com/keboola/go-utils/pkg/wildcards"
-	"github.com/stretchr/testify/assert"
-	etcd "go.etcd.io/etcd/client/v3"
-
-	"github.com/keboola/keboola-as-code/internal/pkg/idgenerator"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/api/gen/buffer"
-	service2 "github.com/keboola/keboola-as-code/internal/pkg/service/buffer/api/service"
-	bufferDependencies "github.com/keboola/keboola-as-code/internal/pkg/service/buffer/dependencies"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/worker/service"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/common/dependencies"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/etcdhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/testhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/testproject"
-)
-
-func TestUploadAndImportE2E(t *testing.T) {
```

Comment:
To je vsetko pokryte uz tu:
https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/test/buffer/worker/worker_test.go#L25-L31